ssl/t1_lib.c

```diff
@@ -693,7 +693,7 @@ int tls1_set_groups(uint16_t **pext, size_t *pextlen,
 
 /* TODO(3.0): An arbitrary amount for now. Take another look at this */
 # define MAX_GROUPLIST   40
-
+# define GROUP_NAME_BUFFER_LENGTH 64
```

Comment:
For cosmetic reasons I'd prefer keeping empty line after the defines. But OK anyway.